### PR TITLE
Hookup import to file storage

### PR DESCRIPTION
**Pull request by Erik9519**
#### Deployment instructions
This branch was merged with the #97 branch and as such should be merged after that has been merged.

#### What have I done
Hooked up the importer to the file storage.
Clear button added in the importer.
Importer now remembers imported files.
(Note: Import and FileStorage both use FileList objects instead of File as FileList objects do not have a constructor and as such I would be unable to re-input the files in the html input component).

#### Testing
JUnit tests.
Use the importer to manually import files and then re-open the importer to verify that the files are remembered.

### CODE REVIEW

**Comment by Extremelyd1** [comment]
Can only be merged after #97 and should be updated in accordance to #97.

File: src/app/import/import.component.ts [hunk 1]
```diff
@@ -198,5 +218,13 @@ export class ImportComponent implements OnInit {
     hasSelected(): boolean {
         return this.selectedModelPreview !== undefined;
     }
+
+    /**
+     * Clear inputted files
+     */
+    clearFiles() {
+        this.pathFile = this.timingFileOne = this.trackFile = this.modelFile = this.timingFileTwo = this.errorFile =
```
[the remark this answers is not in the document]
Comment (Erik9519) [reply src/app/import/import.component.ts]: Changed.

File: src/app/import/import.component.ts [hunk 2]
```diff
@@ -89,7 +93,13 @@ export class ImportComponent implements OnInit {
     }
 
     ngOnInit() {
-        // Intentionally left blank
+        // Reload saved files
```
[the remark this answers is not in the document]
Comment (Erik9519) [reply src/app/import/import.component.ts]: Added comment

File: src/app/backend/fileStorage/fileStorage.service.ts [hunk 3]
```diff
@@ -30,21 +35,24 @@ export class FileStorageService {
...
+     * @param timingFileTwo represents the second timing file
      * @param errorFile represents the error file
      */
-    setFiles(paperPathFile: FileList, modelFile: FileList, trackFile: FileList,
-             timingFile: FileList, errorFile: FileList) {
-        if (paperPathFile !== this.paperPathFile || modelFile !== this.modelFile ||
-            trackFile !== this.trackFile || timingFile !== this.timingFile) {
-            this.paperPathFile = paperPathFile;
-            this.modelFile = modelFile;
+    setFiles(pathFile: FileList, timingFileOne: FileList, trackFile: FileList, modelFile: FileList,
```
[the remark this answers is not in the document]
Comment (Erik9519) [reply src/app/backend/fileStorage/fileStorage.service.ts]: Addressed

File: src/app/backend/fileStorage/fileStorage.service.ts [hunk 4]
```diff
@@ -30,21 +35,24 @@ export class FileStorageService {
...
+             timingFileTwo: FileList, errorFile: FileList) {
+        if (pathFile !== this.pathFile || timingFileOne !== this.timingFileOne || trackFile !== this.trackFile ||
```
[the remark this answers is not in the document]
Comment (Erik9519) [reply src/app/backend/fileStorage/fileStorage.service.ts]: Addressed

File: src/app/backend/fileStorage/fileStorage.service.ts [hunk 5]
```diff
@@ -53,8 +61,9 @@ export class FileStorageService {
     /**
      * Retrieve all input Files
      */
-    getFiles(): {paperPath: FileList, model: FileList, track: FileList, timing: FileList, error: FileList} {
-        return {paperPath: this.paperPathFile, model: this.modelFile, track: this.trackFile,
-                timing: this.timingFile, error: this.errorFile};
+    getFiles(): {pathFile: FileList, timingFileOne: FileList, trackFile: FileList, modelFile: FileList,
+        timingFileTwo: FileList, errorFile: FileList} {
+        return {pathFile: this.pathFile, timingFileOne: this.timingFileOne, trackFile: this.trackFile,
```
[the remark this answers is not in the document]
Comment (Erik9519) [reply src/app/backend/fileStorage/fileStorage.service.ts]: Addressed

File: src/app/import/import.component.scss [hunk 6]
```diff
@@ -58,3 +58,14 @@
     border-top: 1px solid white;
     width: 100%;
 }
+
+.bottom-btn {
+    margin: 0px 10px 10px 10px;
```
[the remark this answers is not in the document]
Comment (Erik9519) [reply src/app/import/import.component.scss]: Addressed

**Pull request merged**